subsys/disk/disk_access_usdhc.c

```diff
@@ -618,8 +618,13 @@ enum usdhc_capability_flag {
 		USDHC_HOST_CTRL_CAP_VS30_MASK,
 	/*!< Support voltage 3.0V */
 	USDHC_SUPPORT_V180_FLAG =
+#if defined(CONFIG_BOARD_MM_SWIFTIO)
```

Comment:
why the ifdef here?

Reply:
In mm_swiftio hardware design, power of sdcard slot is directly connected to 3.3V and can't be switched to 1.8V by VSELECT, so the usdhc and SD card can only use 3.3V signal communication.In order to avoid compatibility problems caused by UHS card using 1.8v signal communication, we can only set usdhc not to support 1.8V on the software.

Reply:
We need to find another way to do this that doesn't require board-specific conditionals in the driver.

Reply:
Do you means to use devicetree or Kconfig for configuration? devicetree looks more suitable, can you give some suggestions? Thanks

Reply:
@MaureenHelm @galak I have changed to use Kconfig to control whether to disable 1.8v high-speed communication. Any board can enable this function according to the needs of the hardware. Please help review whether it is feasible. Thanks!